parameter_store.py

```diff
@@ -0,0 +1,9 @@
+import boto3
+
+def para_store(para_name):
+    ssm = boto3.client('ssm', region_name='us-east-2')
```

Comment:
Please specify the region to Virginia (us-east-1). Remember to move all the parameters to it as well.